### ISSUE TITLE
[ADOC] DAPS ignores width/pdfwidth image attributes and images overflow PDF

### ISSUE THREAD

**Issue by r0ckarong** [openSUSE/daps#501]
### Problem description

Images overflow the PDF content area.

Images at any resolution higher than around 500px width will not fit the rendered PDF and will be cut off. Asciidoctor knows the `width` and `pdfwidth` attributes to control image width in the output formats. DAPS does not honor the `pdfwidth` setting and tries to render the images at maximum resolution which overflows the content area.

![daps_pdf_img1](https://user-images.githubusercontent.com/4920542/56040852-a9828b00-5d37-11e9-95f5-feb8ffd8ef5f.png)
![daps_pdf_img2](https://user-images.githubusercontent.com/4920542/56040853-aa1b2180-5d37-11e9-9c9d-982becf32135.png)

### Expected behavior

The specified content is automatically adjusted to fit into the PDF content area. If not at least the `pdfwidth` argument should work to manually resize images to fit the output.

### Steps to reproduce the behavior

. Define Asciidoc document
. Add image that is wider than ~500px
. Define `pdfwidth=60%` parameter in image macro
. Output to PDF with DAPS

**Comment by r0ckarong** [comment]
The last fix solves the main problem with images in the content. There is the same overflow problem when images are used in a list inside an admonition box.

https://github.com/SUSE/doc-caasp/blame/adoc/adoc/deployment-ecp.adoc#L116

```
.Note down IP/FQDN for nodes
[IMPORTANT]
====
The IP addresses of the generated machines will be displayed in the terraform
output during the cluster node deployment. You need these IP addresses to
deploy {productname} to the cluster.

If you need to find an IP addresses later on you can perform the following steps:

. Log in to {soc} and click on menu:Network[Load Balancers]. Find the one with the string you entered in the terraform configuration above e.g. "testing-lb".
. Note down the "Floating IP". If you have configured a FQDN for this IP, use the hostname instead.
+
image::deploy-loadbalancer-ip.png[pdfwidth=80%]
. Now click on menu:Compute[Instances].
. Switch the filter dropdown to `Instance Name` and enter the string you specified for `stack_name` in the `terraform.tfvars` file.
. Find the Floating IPs on each of the nodes of your cluster.
====
```

![daps_pdf_img3](https://user-images.githubusercontent.com/4920542/56051566-62a08f80-5d4f-11e9-9046-902e5f9d08ce.png)

Same overflow happens in tables:

```
|===
|Text
|image:deploy-loadbalancer-ip.png[pdfwidth=80%]

|image:deploy-loadbalancer-ip.png[pdfwidth=80%]
|Text2
|===
```

![daps_pdf_img4](https://user-images.githubusercontent.com/4920542/56051882-4d783080-5d50-11e9-9c5b-782aa77a4ef5.png)

**Comment by r0ckarong** [comment]
Running into this again trying to use some logos in a file.

The images are all configured with `width` and `pdfwidth` but it doesn't format those.

```
[options="header"]
|===
^|Component ^|URL ^|Version

|image:logo_kubernetes.png[Kubernetes,width=200,pdfwidth=20%,height=70]
|link:https://kubernetes.io/[]
^|*{kube_version}*

|image:logo_crio.svg[CRI-O,width=200,pdfwidth=20%,height=70]
|link:https://cri-o.io/[]
^|*{crio_version}*

|image:logo_cilium.png[Cilium,width=200,pdfwidth=20%,height=70]
|link:https://cilium.io[]
^|*{cilium_version}*

|image:logo_etcd.svg[etcd,width=200,pdfwidth=20%,height=70]
|link:https://github.com/etcd-io/etcd[]
^|*{etcd_version}*

|image:logo_dex.png[dex,width=200,pdfwidth=20%,height=70]
|link:https://github.com/dexidp/dex[]
^|*{dex_version}*

|image:logo_kured.png[Kured,width=200,pdfwidth=20%,height=70] Kured
|link:https://github.com/weaveworks/kured[]
^|*{kured_version}*

|Gangway
|link:https://github.com/heptiolabs/gangway[]
^|*{gangway_version}*
|===
```

![Screenshot_20190510_170047](https://user-images.githubusercontent.com/4920542/57536783-30318480-7345-11e9-8884-1a7b734acb42.png)

**Comment by ghost** [comment]
This should be fixed with 4241968a.
Make sure you use DAPS from the Git repo and your DC file contains:

> ADOC_POST="yes"

If you need this to work with stable DAPS, copy https://raw.githubusercontent.com/openSUSE/daps/develop/daps-xslt/asciidoc/postprocess.xsl into your repo and add the following parameter to the DC file in addition:

> ADOC_POST_STYLE=relative/path/postprocess.xml

(Options documented in https://opensuse.github.io/daps/doc/daps-asciidoc.html#_command_line_options_and_configuration_settings_for_asciidoc .)
